xbmc/xbmc.cpp

```diff
@@ -53,7 +53,7 @@ extern "C" int XBMC_Run(bool renderGUI)
   g_RBP.LogFirmwareVerison();
 #endif
 
-  if (renderGUI && !g_application.CreateGUI())
+  if (renderGUI && !g_application.IsHeadless() && !g_application.CreateGUI())
```

Comment:
if it has the same behavior, it should be a single parameter. from a behavior perspective it is not important what triggered it.
btw: what shared lib sets renderGUI to false? I have searched GitHub and seems to be dead code. nothing sets renderGUi to false.